src/app/components/application/personal-info/parent/parent.component.ts

```diff
@@ -49,6 +49,10 @@ export class ParentComponent implements OnInit {
     parent.isEditing = true;
   }
 
+  canRemoveParent(parent: Parent) {
+    return (parent.contactId !== this.userContactId) && (!this.isParent && this.parents.length > 1);
```

Comment:
This logic also seems maybe wrong, and could do basically the same thing I suggested in the other PR:

```
return parent.contactId !== this.userContactId && this.parents.length > 1;
```